docs/integrations/sources/airtable.md

```diff
@@ -54,10 +54,46 @@ The `Enterprise` level accounts are not supported yet.
 
 ## Data type map
```

Comment:
Worth mentioning somewhere that all the fields are nullable?

Reply:
Done.